react/features/base/react/components/native/SlidingView.tsx

```diff
@@ -70,6 +70,11 @@ interface IState {
  * A generic animated slider view to be used for animated menus.
  */
 export default class SlidingView extends PureComponent<IProps, IState> {
+    /**
+     * Initializes hardwareBackPressSubscription
```

Comment:
Can you end this comment in a period to tame the linter please? Same in the other case.